docs/user/performance.rst

```diff
@@ -13,80 +13,58 @@ Use magnitudes when possible
 
 It's significantly faster to perform mathematical operations on magnitudes (even though your'e still using pint to retrieve them from a quantity object).
 
-.. doctest::
+.. ipython::
+    :verbatim:
```

Comment:
just wondering: why do we use `:verbatim:` here? I think if possible executing would be preferable to keep the docs in sync?

Reply:
I put benchmark with verbatim since it's taking some time to compute. I don't mind putting it back

Reply:
Fixed apostrophe